app/api/helpers/images.py

```diff
@@ -9,8 +16,92 @@ def get_image_file_name():
 
 def uploaded_image(extension='.png', file_content=None):
     filename = get_image_file_name() + extension
-    file_path = current_app.config.get('BASE_DIR') + '/static/uploads/' + filename
+    file_path = app.config.get('BASE_DIR') + '/static/uploads/' + filename
     file = open(file_path, "wb")
     file.write(file_content.split(",")[1].decode('base64'))
     file.close()
     return UploadedFile(file_path, filename)
+
+
+def create_save_resized_image(image_file, basewidth, maintain_aspect, height_size, upload_path,
+                              ext='jpg', remove_after_upload=False):
+    """
+    Create and Save the resized version of the background image
+    :param upload_path:
+    :param ext:
+    :param remove_after_upload:
+    :param height_size:
+    :param maintain_aspect:
+    :param basewidth:
+    :param image_file:
+    :return:
+    """
+    filename = '{filename}.{ext}'.format(filename=get_image_file_name(), ext=ext)
+    image_file = cStringIO.StringIO(urllib.urlopen(image_file).read())
+    im = Image.open(image_file)
+
+    # Convert to jpeg for lower file size.
+    img = Image.new("RGB", im.size, (255, 255, 255))
+    img.paste(im, (0, 0), im)
+
+
+    if maintain_aspect:
+        width_percent = (basewidth / float(img.size[0]))
+        height_size = int((float(img.size[1]) * float(width_percent)))
+
+    img = img.resize((basewidth, height_size), PIL.Image.ANTIALIAS)
+
+    temp_file_relative_path = 'static/media/temp/' + generate_hash(str(image_file)) + get_image_file_name() + '.jpg'
```

Comment:
`static/media/temp` directory may or may not exist. Ensure you create it if needed.